snowshu/adapters/target_adapters/base_target_adapter.py

```diff
@@ -208,15 +199,15 @@ def target_database_is_ready(self) -> bool:
         return self.container.exec_run(
             self.DOCKER_READY_COMMAND).exit_code == 0
 
-    def finalize_replica(self) -> str:
+    def finalize_replica(self) -> None:
         """returns the image name of the completed replica.
```

Comment:
should we update the docstring?

Reply:
Yep, will fix

Reply:
Actually, there was a return that I removed due to it not being used by anything, hence 'None" is valid